clever.go

```diff
@@ -213,7 +214,7 @@ type Section struct {
 	Created      string
 	District     string
 	Grade        string
-	Id           string `json:"id"`
+	Id           string
```

Comment:
So looking at the pattern of the other structs, it seems like you don't need to do:
Blah string `json:"blah"`
since it will automatically know which json field to take.

It looks like it is only needed if the struct name does not match the json name.